release-notes/7.0/preview/api-diff/preview1/Microsoft.AspNetCore.App/7.0-preview1_Microsoft.AspNetCore.Http.Metadata.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.AspNetCore.Http.Metadata
+
+``` diff
+ namespace Microsoft.AspNetCore.Http.Metadata {
++    public interface IFromFormMetadata {
++        string Name { get; }
++    }
++    public interface ISkipStatusCodePagesMetadata
```

Comment:
`IFromFormMetadata` now has a body, but the rerun added `ISkipStatusCodePagesMetadata` without one.

Reply:
The interface is completely empty. So this was a minor tool bug where it doesn't add brackets when no members are found. Adding them manually.